kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -803,6 +804,13 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
              "snake_case to CamelCase conversion")
         ;
 
+    m.def_submodule("ModelPartOperationUtilities", "Free-floating utility functions for model part operations.")
+        .def("CheckValidityOfModelPartsForOperations", &ModelPartOperationUtilities::CheckValidityOfModelPartsForOperations, py::arg("main_model_part"), py::arg("list_of_checking_model_parts"), py::arg("thow_error"))
+        .def("Merge", &ModelPartOperationUtilities::Merge, py::arg("output_model_part_name"), py::arg("main_model_part"), py::arg("model_parts_to_merge"), py::arg("add_neighbours"), py::return_value_policy::reference_internal)
```

Comment:
I would call it `Union` if it is a union

I would not take the python typing as blocker, this can always be ignored

Reply:
done :)